src/AppStream.DurablePatterns/StepsConfig/Entity/StepsConfigEntity.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Newtonsoft.Json;
+
+namespace AppStream.DurablePatterns.StepsConfig.Entity
+{
+    public class StepsConfigEntity : IStepsConfigEntity
```

Comment:
Wht do you thinkg about making it more consistent? Maybe one of:
 - `Step` and `StepsEntity`
 - `StepConfig` and `StepsConfigEntity`
 - `StepConfiguration` and  `StepsConfigurationEntity`
Or what chatGPT suggested when I asked:
 - `StepInfo` `FlowInfoEntity`

Reply:
Sure, first one looks the best to me